speaker_verification/tests/test_sqlmethods.py

```diff
@@ -0,0 +1,89 @@
+import unittest
+
+import pytest
+import numpy as np
+from os.path import join, abspath, dirname
+import pathlib
+import sqlite3
+
+from speaker_verification.sql_utils import(
+    create_db_table,
+    insert_db_row,
+    select_db_row,
+    remove_db_row
+)
+
+table_id = 219944922
+mfcc=np.array([1,2,3])
```

Comment:
Following the good code style is a good practice. Line 17 should be `mfcc = np.array([1,2,3])`